File: IHIS/IHIS.Framework/IHIS.CloudConnector/Contracts/Models/Schs/SCH0201U99BookingDetailInfo.cs
using System;

namespace IHIS.CloudConnector.Contracts.Models.Schs
{
    [Serializable]
    public class SCH0201U99BookingDetailInfo
    {
        private String _bunho;
        private String _suname;
        private String _birth;
        private String _sex;
        private String _hospCode;
        private String _yoyangName;
        private String _tel;
        private String _reserDate;
        private String _reserTime;
        private String _bookingDate;
        private String _linkPatientFlg;
        private String _outHospCode;
        private String _hangmogCode;
        private String _linkEmrFlg;

        public String Bunho
        {
            get { return this._bunho; }
            set { this._bunho = value; }
        }

        public String Suname
        {
            get { return this._suname; }
            set { this._suname = value; }
        }

        public String Birth
        {
            get { return this._birth; }
            set { this._birth = value; }
        }

        public String Sex
        {
            get { return this._sex; }
            set { this._sex = value; }
        }

        public String HospCode
        {
            get { return this._hospCode; }
            set { this._hospCode = value; }
        }

        public String YoyangName
        {
            get { return this._yoyangName; }
            set { this._yoyangName = value; }
        }

        public String Tel
        {
            get { return this._tel; }
            set { this._tel = value; }
        }

        public String ReserDate
        {
            get { return this._reserDate; }
            set { this._reserDate = value; }
        }

        public String ReserTime
        {
            get { return this._reserTime; }
            set { this._reserTime = value; }
        }

        public String BookingDate
        {
            get { return this._bookingDate; }
            set { this._bookingDate = value; }
        }

        public String LinkPatientFlg
        {
            get { return this._linkPatientFlg; }
            set { this._linkPatientFlg = value; }
        }

        public String OutHospCode
        {
            get { return this._outHospCode; }
            set { this._outHospCode = value; }
        }

        public String HangmogCode
        {
            get { return this._hangmogCode; }
            set { this._hangmogCode = value; }
        }

        public String LinkEmrFlg
        {
            get { return this._linkEmrFlg; }
            set { this._linkEmrFlg = value; }
        }

        public SCH0201U99BookingDetailInfo() { }

        public SCH0201U99BookingDetailInfo(String bunho, String suname, String birth, String sex, String hospCode, String yoyangName, String tel, String reserDate, String reserTime, String bookingDate, String linkPatientFlg, String outHospCode, String hangmogCode, String linkEmrFlg)
        {
            this._bunho = bunho;
            this._suname = suname;
            this._birth = birth;
            this._sex = sex;
            this._hospCode = hospCode;
            this._yoyangName = yoyangName;
            this._tel = tel;
            this._reserDate = reserDate;
            this._reserTime = reserTime;
            this._bookingDate = bookingDate;
            this._linkPatientFlg = linkPatientFlg;
            this._outHospCode = outHospCode;
            this._hangmogCode = hangmogCode;
            this._linkEmrFlg = linkEmrFlg;
        }

    }
}

File: MED-CLOUD/med-data/src/main/java/nta/med/data/model/ihis/schs/SCH0201U99BookingDetailInfo.java
package nta.med.data.model.ihis.schs;

import java.math.BigDecimal;
import java.util.Date;

public class SCH0201U99BookingDetailInfo {
	private String bunho                  ;
	private String suname                 ;
	private Date birth                  ;
	private String sex                    ;
	private String hospCode              ;
	private String yoyangName            ;
	private String tel                    ;                                        
	private Date reserDate             ;
	private String reserTime             ;
	private String bookingDate           ;
	private String outHospCode          ;
	private String hangmogCode           ;
	private BigDecimal linkEmrFlg           ;
	public SCH0201U99BookingDetailInfo(String bunho, String suname, Date birth, String sex, String hospCode,
			String yoyangName, String tel, Date reserDate, String reserTime, String bookingDate, String outHospCode, String hangmogCode, BigDecimal linkEmrFlg) {
		super();
		this.bunho = bunho;
		this.suname = suname;
		this.birth = birth;
		this.sex = sex;
		this.hospCode = hospCode;
		this.yoyangName = yoyangName;
		this.tel = tel;
		this.reserDate = reserDate;
		this.reserTime = reserTime;
		this.bookingDate = bookingDate;
		this.outHospCode = outHospCode;
		this.hangmogCode = hangmogCode;
		this.linkEmrFlg = linkEmrFlg;
	}
	public String getBunho() {
		return bunho;
	}
	public void setBunho(String bunho) {
		this.bunho = bunho;
	}
	public String getSuname() {
		return suname;
	}
	public void setSuname(String suname) {
		this.suname = suname;
	}
	public Date getBirth() {
		return birth;
	}
	public void setBirth(Date birth) {
		this.birth = birth;
	}
	public String getSex() {
		return sex;
	}
	public void setSex(String sex) {
		this.sex = sex;
	}
	public String getHospCode() {
		return hospCode;
	}
	public void setHospCode(String hospCode) {
		this.hospCode = hospCode;
	}
	public String getYoyangName() {
		return yoyangName;
	}
	public void setYoyangName(String yoyangName) {
		this.yoyangName = yoyangName;
	}
	public String getTel() {
		return tel;
	}
	public void setTel(String tel) {
		this.tel = tel;
	}
	public Date getReserDate() {
		return reserDate;
	}
	public void setReserDate(Date reserDate) {
		this.reserDate = reserDate;
	}
	public String getReserTime() {
		return reserTime;
	}
	public void setReserTime(String reserTime) {
		this.reserTime = reserTime;
	}
	public String getBookingDate() {
		return bookingDate;
	}
	public void setBookingDate(String bookingDate) {
		this.bookingDate = bookingDate;
	}
	public String getOutHospCode() {
		return outHospCode;
	}
	public void setOutHospCode(String outHospCode) {
		this.outHospCode = outHospCode;
	}
	public String getHangmogCode() {
		return hangmogCode;
	}
	public void setHangmogCode(String hangmogCode) {
		this.hangmogCode = hangmogCode;
	}
	public BigDecimal getLinkEmrFlg() {
		return linkEmrFlg;
	}
	public void setLinkEmrFlg(BigDecimal linkEmrFlg) {
		this.linkEmrFlg = linkEmrFlg;
	}
	

}
